tests/it.xsemantics.dsl.tests/tests_input_files/header_test.xsemantics

```diff
@@ -0,0 +1,31 @@
+import it.xsemantics.example.fj.fj.Expression
+import it.xsemantics.example.fj.fj.Type
+import it.xsemantics.example.fj.fj.Class
+import it.xsemantics.example.fj.fj.New
+
+import it.xsemantics.example.fj.util.FjTypeUtils
+import it.xsemantics.example.fj.validation.AbstractFJJavaValidator
+import java.util.Collections
+import java.util.LinkedList
+import java.util.List
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.EcoreUtil2
+
+system it.xsemantics.test.fj.first.FjFirstTypeSystem
```

Comment:
Please use another name instead of FjFirstTypeSystem or tests that run the headless xsemantics compiler will fail because there's an existing system with the same name.